site/content/blog/release-0.30.0.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Release 0.30.0 is out!"
+date: 2023-02-20T10:53:49+11:00
+draft: false
+---
+
+`rust-bitcoin` version 0.30.0 is out now.
+
+<!--more-->
+
+This is a rather large release so we decided to write an update guide for you guys. If this guide is
+not useful or lacking in some way please do let us know so we can do better.
+
+First a little excuse for why this is going to be so painful. We have, and do, try to deprecate
+things if we make API breaking changes. We do this using
```

Comment:
Sorry for last minute review here.

nit: awkward sentence. I suggest replacing this with
```
First, a little excuse for why this is going to be so painful. We try to deprecate things
when we make API breaking changes, using
```